resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml

```diff
@@ -45,6 +45,13 @@ spec:
                     type: string
                   prom_token:
                     type: string
+                  system_metrics_collection:
+                    type: boolean
+                    default: false
+                  system_metrics_index:
```

Comment:
In the description of the PR it states the metrics index and profile are defaulted to X. I assume that default is in the kube-burner code. Would it make sense to default it hear as well so it's more obvious to the end user?